analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/CookieShouldBeHttpOnlyTest.cs

```diff
@@ -99,5 +98,8 @@ public void CookiesShouldBeHttpOnly_WithWebConfigValueSetToFalse()
         private static IEnumerable<MetadataReference> GetAdditionalReferences_NetCore() =>
             NuGetMetadataReference.MicrosoftAspNetCoreHttpFeatures(Constants.NuGetLatestVersion);
 #endif
+
+        internal static IEnumerable<MetadataReference> AdditionalReferences =>
```

Comment:
nitpick internal should be above private (you can make this change in your next PR though)

Reply:
I realize now that the private one is inside a preprocessor directive, so it doesn't apply